### ISSUE TITLE
Apache proxy failure with auth_type config

### ISSUE THREAD

**Issue by pma-import** [phpmyadmin/phpmyadmin#10694]
This report comes directly from https://stackoverflow.com/questions/24372485/phpmyadmin-authentication-issue-with-apache-proxy

phpMyAdmin works using Apache proxy if `Authentication type` is set to `http`.  However, it does not work if `Authentication type` is set to `config`.  By not work I mean I get a timeout message.

I have two Apache servers: one in a public, internet accessible zone (PAZ) and the other in a private restricted zone (RZ).  The proxy config is in the PAZ Apache and the phpMyAdmin and MySQL are installed on the RZ server.

This is the proxy setup on the PAZ Apache

    ProxyPass /phpMyAdmin-4.2.2-english/ https://restricted-zone-server/phpMyAdmin-4.2.2-english/

---
* **Original URL**: https://sourceforge.net/p/phpmyadmin/bugs/4583/
* **Original author**: ibennetch
* **Found in version**: 4.2.2

**Comment by udan11** [comment]
I can't reproduce this bug.

The way I tested:
* created two virtual machines (using Vagrant); both ran Ubuntu 14.04.2 LTS;
* on `vm1` I installed Apache with mod_proxy; on `vm2` installed the LAMP stack;
* on `vm1` configured mod_proxy to redirect to `vm2`; on `vm2` banned my laptop using `iptables`;
* tried accessing `vm1` from my laptop, everything worked fine (tried `cookie` and `config` authentication types)

He may have not set `ProxyPassReverse` or misconfigured something else which resulted in a time out. The original question on StackOverflow is unavailable so I can't figure out what his problem could be.
